RELEASE_PROCESS.md

```diff
@@ -0,0 +1,64 @@
+# Our Release Process
+
+## Notes
+
+BigchainDB follows
+[the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme)
+(i.e. MAJOR.MINOR.PATCH),
+which is almost identical
+to [regular semantic versioning](http://semver.org/), but there's no hyphen, e.g.
+
+- `0.9.0` for a typical final release
+- `4.5.2a1` not `4.5.2-a1` for the first Alpha release
+- `3.4.5rc2` not `3.4.5-rc2` for Release Candidate 2
+
+**Note 1:** For Git tags (which are used to identify releases on GitHub), we append a `v` in front. For example, the Git tag for version `2.0.0a1` was `v2.0.0a1`.
+
+**Note 2:** For Docker image tags (e.g. on Docker Hub), we use longer version names for Alpha, Beta and Release Candidate releases. For example, the Docker image tag for version `2.0.0a1` was `2.0.0-alpha`.
+
+We use `0.9` and `0.9.0` as example version and short-version values below. You should replace those with the correct values for your new version.
+
+We follow [BEP-1](https://github.com/bigchaindb/BEPs/tree/master/1), which is our variant of C4, the Collective Code Construction Contract, so a release is just a [tagged commit](https://git-scm.com/book/en/v2/Git-Basics-Tagging) on the `master` branch, i.e. a label for a particular Git commit.
+
+The following steps are what we do to release a new version of _BigchainDB Driver_.
```

Comment:
Since there is more than one BigchainDB driver, maybe write "a new version of the BigchainDB Python Driver."